src/rnp/rnpcfg.h

```diff
@@ -125,6 +129,8 @@
     "kg-primary-sphincsplus-param" /* 128f, 128s, 192f, 192s, 256f, 256s */
 #define CFG_KG_SUBKEY_SPHINCSPLUS_PARAM \
     "kg-subkey-sphincsplus-param" /* 128f, 128s, 192f, 192s, 256f, 256s */
+#define CFG_KG_SUBKEY_2_SPHINCSPLUS_PARAM \
```

Comment:
Looks like I missed this during the previous review, but could you please but these under the PQC define?